packages/webdriverio/tests/commands/browser/uploadFile.test.js

```diff
@@ -1,9 +1,8 @@
 import path from 'path'
 import archiver from 'archiver'
 
-import { remote } from '../../../src'
-
 describe('uploadFile', () => {
+    const { remote } = require('../../../src')
```

Comment:
Why is it here other way around? Because in this test we actually want to have the mocked version and in others we don't?

Reply:
I'll try to explain how it works after my changes in case we keep `remote` import as it was before.

When `remote` is imported all the commands are loaded from file system.
If `fs` was fully mocked before no commands are loaded.
If `fs` was not mocked and `remote` is imported, then there is no way to mock https://github.com/webdriverio/webdriverio/blob/3aa49945eba252aa70641088c8bf816129b49ccd/packages/webdriverio/src/commands/browser/uploadFile.js#L25 because commands were loaded before `fs` is mocked.

So I decided to mock fs, restore readdirSync and then import everything else. I can dig a bit more into it in the evening.

If you have any hints on how can this be improved please let me know